menu/menu_displaylist.c

```diff
@@ -924,9 +950,14 @@ static int menu_displaylist_parse_playlist(menu_displaylist_info_t *info,
           *   no further action is necessary */
 
          if (string_is_empty(entry->label))
+         {
             fill_short_pathname_representation(menu_entry_label, entry->path, sizeof(menu_entry_label));
+         }
```

Comment:
Just a matter of coding style - we don't use braces when the statement occupies only a single line
(http://docs.libretro.com/development/coding-standards/)